qiskit/circuit/library/standard_gates/h.py

```diff
@@ -109,6 +109,8 @@ class CHGate(ControlledGate):
     in the :math:`|1\rangle` state.
 
     **Circuit symbol:**
+    
```

Comment:
```suggestion

```
One lint failure due to extra whitespace on L112: https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=31402&view=logs&j=f0ab2c38-58fa-55ad-8e8b-d3984171f737&t=980625fc-4a64-50e4-7ca0-679c2a4bbda8&l=15 .